src/library/FOSSBilling/Monolog.php

```diff
@@ -51,7 +51,7 @@ public function __construct(protected ?\Pimple\Container $di)
             $this->logger[$channel]->pushHandler($stream);
 
             $formatter = new LineFormatter($this->outputFormat, $this->dateFormat, true, true, true);
-            $this->logger[$channel]->getHandlers()[0]->setFormatter($formatter);
+            $this->logger[$channel]->getHandlers()[0]->setFormatter($formatter); // @phpstan-ignore-line
```

Comment:
// @phpstan-ignore-line Should be the absolute exeption, why is it used here? I think a comment explaining that would be good.